cmd/soroban-cli/src/commands/config/alias.rs

```diff
@@ -0,0 +1,107 @@
+use std::{
+    collections::HashMap,
+    fs::{self, create_dir_all, OpenOptions},
+    io::Write,
+    path::PathBuf,
+};
+
+use serde::{Deserialize, Serialize};
+use stellar_strkey::DecodeError;
+
+use crate::commands::config;
+
+use super::Args;
+
+#[derive(Serialize, Deserialize, Default)]
+pub struct Data {
+    ids: HashMap<String, String>,
+}
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Json(#[from] serde_json::Error),
+    #[error(transparent)]
+    Io(#[from] std::io::Error),
+    #[error("cannot access config dir for alias file")]
+    CannotAccessConfigDir,
+    #[error("cannot parse contract ID {0}: {1}")]
+    CannotParseContractId(String, DecodeError),
+    #[error(transparent)]
+    Config(#[from] config::Error),
+}
+
+impl Args {
+    pub fn load(&self, alias: &str) -> Result<Option<Data>, Error> {
```

Comment:
```suggestion
    pub fn load_alias_data(&self, alias: &str) -> Result<Option<Data>, Error> {
```
Now this is in `super::Args`, it's too general. Also does this need to be public?

Reply:
nice catch!